sdk/php/runtime/main.go

```diff
@@ -72,14 +96,31 @@ func (m *PhpSdk) CodegenBase(
 	}
 
 	base := dag.Container().
-		From(fmt.Sprintf("%s@%s", PhpImage, PhpDigest)).
-		WithExec([]string{"apk", "add", "git", "openssh", "curl"}).
-		WithFile("/usr/bin/composer", dag.Container().From(ComposerImage).File("/usr/bin/composer")).
-		WithMountedCache("/root/.composer", dag.CacheVolume(fmt.Sprintf("composer-%s", PhpImage))).
+		From(fmt.Sprintf("php:%s", m.PhpVersion)).
+		WithExec([]string{"apk", "add", "git", "openssh", "curl", "jq"}).
+		WithFile("/usr/bin/composer", dag.Container().From(ComposerImage).File("/composer")).
+		WithMountedCache("/root/.composer", dag.CacheVolume(fmt.Sprintf("composer-%s", m.PhpVersion))).
 		WithEnvVariable("COMPOSER_HOME", "/root/.composer").
 		WithEnvVariable("COMPOSER_NO_INTERACTION", "1").
 		WithEnvVariable("COMPOSER_ALLOW_SUPERUSER", "1")
 
+	if len(m.PhpExtInstall) > 0 {
+		for _, ext := range m.PhpExtInstall {
+			base = base.
+				WithExec([]string{"docker-php-ext-install", ext})
```

Comment:
The `docker-php-ext-install` script cannot be automated easily:

> Remember, you must install dependencies for your extensions manually. If an extension needs custom configure arguments, you can use the docker-php-ext-configure script

https://hub.docker.com/_/php#php-core-extensions